specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -298,6 +357,36 @@
           }
         }
       }
+    },
+    "CanceledSubscriptionId": {
+      "description": "Canceled Subscription Id",
+      "properties": {
+        "value": {
+          "description": "Canceled Subscription Id",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RenamedSubscriptionId": {
+      "description": "Renamed Subscription Id",
+      "properties": {
+        "value": {
+          "description": "Renamed Subscription Id",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "getSubscriptionName": {
```

Comment:
why getSubscriptionName instead of just subscriptionName

Reply:
Changed the name